lib/wab/io/engine.rb

```diff
@@ -13,19 +13,18 @@ class Engine
       # tcnt:: processing thread count
       def initialize(shell, tcnt)
         @shell = shell
-        @last_rid = 0
```

Comment:
This can not be a local variable later. It is the number of requests made and used to create sequence numbers.

Reply:
where is the instance_variable accessed later?

Reply:
In the `request` function inside the synchronize block.